include/eve/arch/riscv/spec.hpp

```diff
@@ -0,0 +1,39 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch/riscv/predef.hpp>
+
+#include <cstddef>
+
+//==================================================================================================
+// Register count
+//==================================================================================================
+#if defined(EVE_HW_RISCV_SVE)
```

Comment:
I don't think this is the correct macro

Reply:
Do you mean that I should use `EVE_INCLUDE_RISCV_HEADER`? Fixed, and removed for now define `EVE_HW_RISCV_SVE` define, as it is not used now.